assets/js/theme/product/reviews.js

```diff
@@ -89,6 +89,10 @@ export default class {
             selector: '[name="revtitle"]',
             validate: 'presence',
             errorMessage: safeString(this.context.reviewSubject),
+        }, {
+            selector: '[name="revfromname"]',
```

Comment:
@Dpschu2 I think you have meant `[name="revformname"]`, not `[name="revfromname"]` ?

Reply:
already got answer for this, all is good 👍